Add desktop version for the project:

norbiboy1 opened the pull request:
- desktop version design with alignment all of the sections
- use grid and flexbox
- use media queries for the webpage
- validate contact form

Commit:
Add:
- desktop version design with alignment all of the sections
- use grid and flexbox
- use media queries for the webpage
- validate contact form

GzimAsani commented on portfolio.html:
```diff
@@ -8,6 +8,284 @@
+    <!-- Desktop version headline -->
+    <section class="desktop-headline">
+        <div class="desktop-headline-first-text">
+            <h2 class="desktop-headline-heading">Hey there. I’m Norbi<br><span>I’m a software developer</span></h2>
```
- [ ] This section is being rendered using position absolute, this is not recommended, please use only flex or grid
![image](https://user-images.githubusercontent.com/62705947/186935835-452c5c88-f43c-4814-bf4d-c5f0bb2c35af.png)

GzimAsani commented on portfolio.html:
```diff
@@ -8,6 +8,284 @@
+
+        <div class="desktop-normal-button">
+            <button type="button" class="btn2">
+                <img src="./images/NormalButton2/Secondary/Disabled.png" class="desktop-disabled-button" alt="Arrow down icon">
```
- [ ] This button has two circles, kindly fix it
![image](https://user-images.githubusercontent.com/62705947/186936009-0f5389f2-bd5f-456d-ac7f-d8e16d7e8f70.png)

GzimAsani commented on portfolio.html:
```diff
@@ -8,6 +8,284 @@
+    <section class="desktop-work">
+        <div class="desktop-recent-works">
+            <p class="desktop-primary-text">
+                My Recent Works
```
- [ ] In this section, the side borders are missing on all the cards kindly update them
![image](https://user-images.githubusercontent.com/62705947/186936259-2244d3f8-54ed-4050-abaa-46fc7b6096ee.png)

GzimAsani replied:
- [ ] Kindly add some space here, as in the design
![image](https://user-images.githubusercontent.com/62705947/186936542-6827a24e-4009-49fe-aa1b-ca09c8547513.png)

GzimAsani replied:
- [ ] You have used position absolute here to layout the elements, that is not recommended, please use flexbox or grid
![image](https://user-images.githubusercontent.com/62705947/186936838-a38ab9e9-d573-4ad2-9b8e-e4ac215d59e0.png)

GzimAsani commented on portfolio.html:
```diff
@@ -8,6 +8,284 @@
+                        <li class="html">html</li>
+                    </ul>
+                    <div class="desktop-button-frame">
+                        <button type="button" class="desktop-btn-green">See Project</button>
```
- [ ] Kindly implement the **hover** and after **pressing** and **exact** color of all the buttons on your page, you can check them by clicking on the figma on the button
![image](https://user-images.githubusercontent.com/62705947/186937115-09df4f23-bf45-408a-b304-e284a7aeadea.png)

GzimAsani commented on portfolio.html:
```diff
@@ -8,6 +8,284 @@
+        <img src="./images/desktop_version/frame_headline/illustration-about-me-2-desktop@2x.png" class="illustration-about-me-2" alt="Illustration icon-2">
+        <img src="./images/desktop_version/frame_headline/illustration-about-me-1-desktop@2x.png" class="illustration-about-me-1" alt="Illustration icon-1">
+        <div class="desktop-about-me-header">
+            <h2 class="desktop-about-me-primary-text">About me</h2>
```
- [ ] You have used position absolute to center the elements, which is not recommended, kindly use flexbox or grid instead
![image](https://user-images.githubusercontent.com/62705947/186937492-5a304e74-2b20-4dbb-856f-8c1da8a3e2d6.png)

GzimAsani commented on portfolio.html:
```diff
@@ -8,6 +8,284 @@
+
+    <section class="desktop-form">
+        <div class="desktop-form-primary-text">
+            <h2 class="desktop-form-text">I'm always interested in hearing about new projects, so if you'd <br> like to chat please get in touch.</h2>
```
- [ ] Check the border colors of the form and adjust them according to the desgin
![image](https://user-images.githubusercontent.com/62705947/186937833-a490f44e-6cff-43ac-ab4a-33bd3e67241b.png)

GzimAsani commented on portfolio.html:
```diff
@@ -8,6 +8,284 @@
+
+    <!-- Background images first section -->
+    <div class="background-images-desktop">
+        <img src="./images/desktop_version/frame_headline/Frame.png" class="desktop-frame" alt="Frame group of icon">
+        <img src="./images/desktop_version/frame_headline/Group-38.png" class="desktop-group-38" alt="Group 38 image">
+        <img src="./images/desktop_version/frame_headline/Group2.png" class="desktop-group-2" alt="Group 2 image">
+        <img src="./images/desktop_version/frame_headline/Frame-big.png" class="desktop-frame-big" alt="Frame big icon">
+        <img src="./images/desktop_version/frame_headline/Group-37.png" class="desktop-group-37" alt="Group 37 image">
+        <img src="./images/desktop_version/frame_headline/Frame-middle.png" class="desktop-frame-middle" alt="Frame middle icon">
+        <img src="./images/desktop_version/frame_headline/Frame-right.png" class="desktop-frame-right" alt="Frame right icon">
+        <img src="./images/desktop_version/frame_headline/Group-35.png" class="desktop-group-35" alt="Group 35 image">
```
- [ ] Images should be introduced as background images in the CSS file of the particular container that you use, and not through HTML. Please be reminded that you should use HTML only for images that have an important meaning in your app(similar to project images), but not for background images

GzimAsani reviewed (changes requested):
### Hello @norbiboy1  :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job  with the styling, looks good  :+1: 
 
 ## Required Changes 

- [ ] Usage of position absolute is not recommended, kindly use flexbox and grid
- [ ] Check the comments below :arrow_double_down: 




 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Changes for the desktop version:
- display flex and delete position: absolute on sections;
- add background-image and delete all the image sources;
- add active, hover, enable changing colors for all the buttons;
- fix the button disable icon(1 circle)

Nemwel-Boniface commented on portfolio.html:
```diff
@@ -8,6 +8,259 @@
+    </section>
+
+    <!-- Desktop version work section -->
+    <section class="desktop-work">
```
- [ ] It appears that after the new changes were introduced the view of this section is distorted. I am viewing the project at 100% zoom level. Kindly center this section so that it matches the chosen Figma design.

**How I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/187041273-6a44ad6d-1e3a-430d-832b-7c756475d8d8.png)

**The desired design:**

![image](https://user-images.githubusercontent.com/86318284/187041314-871e3472-a825-4a5d-8436-9224c825a40e.png)

Nemwel-Boniface commented on portfolio.html:
```diff
@@ -8,6 +8,259 @@
+                    <img src="./images/mobile_version/Card/Img_Plaholder.png" class="desktop-image-cover" alt="image">
+                    <div class="desktop-text-card-1">
+                        <h2 class="desktop-card-1-heading">Multi-Post Stories <br> Gain+Glory</h2>
+                    <ul class="desktop-list-languages">
+                        <li class="ruby">Ruby on rails</li>
+                        <li class="css">css</li>
+                        <li class="javascript">JavScript</li>
+                        <li class="html">html</li>
+                    </ul>
```
- [ ] It appears that after the new changes were introduced the view of this section is distorted. I am viewing the project at 100% zoom level. Kindly center this section so that it matches the chosen Figma design.

**How I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/187041376-d9a3996d-be30-4c61-9c7f-3b6c3fefbf95.png)

**The desired design:**

![image](https://user-images.githubusercontent.com/86318284/187041363-072bf072-2e18-4039-b85a-f5c6a3fd7797.png)

Nemwel-Boniface reviewed (changes requested):
## Hello @norbiboy1           :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job addressing some of the changes that were requested by a previous reviewer :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif)
## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
I add a larger width for the desktop-list-languages.
I add width for the headline section.

Ismailco commented on portfolio.html:
```diff
@@ -8,6 +8,259 @@
     <link href="portfolio.css" rel="stylesheet">
 </head>
 <body>
```
- [ ] As the previous reviewer mentioned your page is not centered. good job 👍🏼 implementing some of the changes however kindly make sure to center all the items on your page so it will reflect the Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220827_185829](https://user-images.githubusercontent.com/25227022/187042680-66ec68a2-11ea-4fe2-b4c4-84b1c0cab0b6.png)|![](https://user-images.githubusercontent.com/86318284/187041314-871e3472-a825-4a5d-8436-9224c825a40e.png)|

Ismailco reviewed (changes requested):
### Hi @norbiboy1, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Add maximum width for html container to look responsive my webpage.

victorokoroji commented on portfolio.css:
```diff
@@ -2108,4 +1064,805 @@ input[type=email] {
+    justify-content: center;
+    padding: 80px 0;
+    gap: 48px;
+    width: 2063px;
```
- [ ] Kindly note that `_**About me**_` section for the desktop view is currently not centered. This is attributed to the width you specified for the section ( width: 2063px; ). Kindly remove the width and the section will be centralized just as the previous reviewer suggested 😄 

![image](https://user-images.githubusercontent.com/63473584/187044415-d2e32ef5-a3fe-47e3-bd5f-bde4f9dcffb0.png)

victorokoroji commented on portfolio.css:
```diff
@@ -2108,4 +1064,805 @@ input[type=email] {
+    align-items: center;
+    padding: 300px 0;
+    gap: 48px;
+    width: 2063px;
```
- [ ] Kindly note that the `_**Work Section**_` section for the desktop view is currently not centered. This is attributed to the width you specified for the section ( width: 2063px; ). Kindly remove the width and the section will be centralized just as the previous reviewer suggested 😄 

![image](https://user-images.githubusercontent.com/63473584/187044519-184dd2f7-1d6a-4f72-b6c9-025d02e2f3eb.png)

victorokoroji commented on portfolio.css:
```diff
@@ -2108,4 +1064,805 @@ input[type=email] {
+    justify-content: center;
+    padding: 80px 0;
+    gap: 24px;
+    width: 2063px;
```
- [ ] Kindly note that _**Contact**_ section for the desktop view is currently not centered. This is attributed to the width you specified for the section ( width: 2063px; ). Kindly remove the width and the section will be centralized 👍🏽 .

![image](https://user-images.githubusercontent.com/63473584/187044592-a9f52b8d-8a56-41b0-ac46-95d5baa1cfc7.png)

victorokoroji commented on portfolio.css:
```diff
@@ -2108,4 +1064,805 @@ input[type=email] {
+    align-items: center;
+    padding: 160px 0;
+    gap: 24px;
+    width: 2063px;
```
- [ ]  Kindly note that **_`Headline section`_** for the desktop view is currently not centered. This is attributed to the width you specified for the section ( width: 2063px; ). Kindly remove the width and the section will be centralized 👍🏽 .
![image](https://user-images.githubusercontent.com/63473584/187044653-b521f492-ab51-498e-a016-cc29803dad19.png)

victorokoroji reviewed (changes requested):
Hi @norbiboy1 , 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![](https://camo.githubusercontent.com/56bcd754643682fcc078b66eaa7278dead6c9c5013595c875b4ddcbe0f0ef0aa/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d2e676966)

## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Make my website looks professionaly by adding:
1. Media queries for each device - mobile and desktop.
2. Responsiveness.
3. Design and style.
